docs/fixes/2026-05-25-store-hook-missing-backend-role-assumption.md

```
@@ -0,0 +1,155 @@
+    outputs:
+      my_output: .my_output
+```
+
+```bash
+atmos terraform apply my-component -s my-stack
+```
+
+Apply succeeds (Atmos handles the full credential chain), but the
+after-apply hook fails:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)